src/solver/application.h

```diff
@@ -40,6 +40,13 @@ class Application : public Yuni::IEventObserver<Application, Yuni::Policy::Singl
     */
     void prepare(int argc, char* argv[]);
 
+    /*!
+    ** \brief Check validity of adequacy patch parameters
+    **
+    ** \return False if the operation failed.
+    */
+    void prepareAdqPatchParams();
+
```

Comment:
1. Replace `prepare` with `check`
2. Notice how most parameters are in `adqPatchParams`. This strongly suggest that we should create a member function in class `AdequacyPatch::AdqPatchParams`